pgen-native/Makefile

```diff
@@ -0,0 +1,25 @@
+CFLAGS := -Wall -g
+
+OBJS := parser.o \
+	tokenizer.o \
+	grammar1.o \
+	grammar.o \
+        bitset.o \
```

Comment:
Whoops, looks like despite my efforts, emacs still *really* wants to put tabs in my files.